images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -236,32 +242,32 @@ function Get-PowerShellAzureModules {
 
     [Array] $azInstalledModules = Get-ChildItem -Path "C:\Modules\az_*" -Directory | ForEach-Object { $_.Name.Split("_")[1] }
     if ($azInstalledModules.Count -gt 0) {
-        $result += "Az: $($azInstalledModules -join ', ')"
+        $result += [ToolVersionsListNode]::new("Az", $($azInstalledModules), '^\d+', "Inline")
     }
 
     [Array] $azureInstalledModules = Get-ChildItem -Path "C:\Modules\azure_*" -Directory | ForEach-Object { $_.Name.Split("_")[1] } | ForEach-Object { if ($_ -eq $defaultAzureModuleVersion) { "$($_) (Default)" } else { $_ } }
     if ($azureInstalledModules.Count -gt 0) {
-        $result += "Azure: $($azureInstalledModules -join ', ')"
+        $result += [ToolVersionsListNode]::new("Azure", $($azureInstalledModules), '^\d+', "Inline")
```

Comment:
I think we should use `^\d+\.\d+` regex for Azure modules

Reply:
agree, corrected